beta-src/src/components/ui/WDPhaseUI.tsx

```diff
@@ -93,6 +100,7 @@ const WDPhaseUI: React.FC = function (): React.ReactElement {
           gamePhase={gamePhase}
           gameSeason={gameSeason}
           gameYear={gameYear}
+          isPaused={user?.member.votes.includes("Pause") || false}
```

Comment:
I'm not sure this is correct. Does the game pause just because the user votes "Pause"? I don't think so.

You probably need to check something like `status.status === "Paused"`, but I'm not sure.

Reply:
you were right, I updated it in the latest commit. It was tricky to find the right property